app/assets/stylesheets/govuk_publishing_components/components/_checkboxes.scss

```diff
@@ -27,7 +27,7 @@
 
 .gem-c-checkboxes {
   // this is complex but needed to override the govuk-frontend styles in
-  // https://github.com/alphagov/govuk-frontend/blob/b8058640b9602ecb6e1f66f887553190cbae7b46/src/components/hint/_hint.scss#L16
+  // https://github.com/alphagov/govuk/blob/b8058640b9602ecb6e1f66f887553190cbae7b46/src/components/hint/_hint.scss#L16
```

Comment:
Also not sure this one should have been changed from govuk-frontend to govuk.

Reply:
Fixed, missed when replacing. Good spot.